nengo/params.py

```diff
@@ -445,7 +445,7 @@ def coerce_ndarray(self, instance, ndarray):  # noqa: C901
 
     @property
     def coerce_defaults(self):
-        return not ("n_neurons" in self.shape or "dimensions" in self.shape)
+        return not any(is_string(dim) for dim in self.shape)
```

Comment:
`"..."` and `"*"` are valid shape strings that don't need to disable the default coercion

Reply:
right, fixed it